src/test/java/fi/aalto/cs/apluscourses/model/SubmissionStatusUpdaterTest.java

```diff
@@ -67,13 +69,14 @@ public void testSubmissionStatusUpdater() throws InterruptedException {
 
     assertEquals("The submission results are not fetched anymore after feedback is available",
         3, dataSource.getSubmissionResultFetchCount());
-    verify(notifier).notify(any(FeedbackAvailableNotification.class), isNull());
+    verify(notifier).notify(any(FeedbackAvailableNotification.class), any(Project.class));
```

Comment:
Nit: could check that the `Project` object is the same that was given to the `SubmissionStatusUpdater`. But this is a truly minor thing.